CHANGELOG.md

```diff
@@ -9,7 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Skipping all validation and defaulting for resources with CAPI release label
+- Skipping all validation and defaulting for resources with CAPI release label.
+- Skipping all validation and defaulting for resources with CAPI watch filter label.
```

Comment:
```suggestion
- Skipping all validation and defaulting for resources with the release label containing a release `>=` than the first GiantSwarm release with CAPI controllers.
- Skipping all validation and defaulting for resources with the CAPI watch filter label.
```

Reply:
I just added a dot on the end of that sentence :smile:

Reply:
I know, I know, but after your changes it wasn't clear to me what the first sentence was trying to explain.